packages/wp-source/src/libraries/handlers/taxonomy.ts

```diff
@@ -69,7 +72,15 @@ const taxonomyHandler = ({
     isArchive: true,
     isTaxonomy: true,
     [`is${capitalize(firstPageData.taxonomy)}`]: true
-  });
+  } as TaxonomyData);
```

Comment:
adding `as TaxonomyData` seems like the best (only?) way to ensure typesafety when doing `Object.assign`

Reply:
not relevant anymore, we decided to forgo the typesafety for the time being in this case until we do a proper refactor of wp-source :)